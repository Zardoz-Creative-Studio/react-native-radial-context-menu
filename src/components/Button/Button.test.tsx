import * as React from "react";
import renderer from "react-test-renderer";
import { Button, IButtonProps } from "./Button";

describe("Button", () => {
  it("renders the component correctly", () => {
    const props: IButtonProps = {
      active: false,
      backgroundColor: "#3f3f3f",
      buttonDiameter: 40,
      vibrate: false,
      vibrateDuration: 50,
      x: 0,
      y: 0,
    };

    expect(renderer.create(<Button {...props} />).toJSON()).toMatchSnapshot();
  });
});
