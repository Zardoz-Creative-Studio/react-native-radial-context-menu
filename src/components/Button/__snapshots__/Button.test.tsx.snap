// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders the component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#3f3f3f",
      "borderRadius": 999,
      "height": 40,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "transform": Array [
        Object {
          "scale": 1,
        },
      ],
      "width": 40,
      "zIndex": 999,
    }
  }
>
  <View />
</View>
`;
